src/components/Forms/formStatusHook.js

```diff
@@ -0,0 +1,79 @@
+import { useState, useEffect } from 'react';
+import { faCheck, faSpinner, faTimes } from '@fortawesome/free-solid-svg-icons';
+import axios from 'axios';
+
+export function useFormStatus(
+  apiLink = '',
+  formData = {},
+  buttonStatus = { text: 'Submit', icon: null, class: '' },
+  formsStatus = { isSuccess: false, isLoading: false, isError: false }
```

Comment:
Currently, we are just passing in empty objects for this. It's probably better to remove the argument if it's not being used.